packages/scandipwa/src/component/Field/Field.component.js

```diff
@@ -190,7 +190,7 @@ export class Field extends PureComponent {
             type,
             setRef,
             attr,
-            attr: { id = '' } = {},
+            attr: { id = '', checked = null, value = '' } = {},
```

Comment:
point of setting it to be default null? So if undefined then become null? In this case it will take isChecked value depending on disableButton boolean.

Reply:
Yeah you are right! Done the mods